week-3/Homework/mandatory/7-password-validator.js

```diff
@@ -22,8 +22,25 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {
+function checkPassword(password,index,arr) {
+  let passUppercase = /[A-Z]/;
+  let passLowerercase = /[a-z]/;
+  let passNumber = /[0-9]/;
+  let passSymbols = /[!#$%.]/;
```

Comment:
Really nice usage of regexes! 💪

Reply:
thank you

Reply:
You broke down the regex expression instead of leaving it in one line and s it makes it more readable. Good job!